packages/server/src/Server.ts

```diff
@@ -62,6 +66,9 @@ export class Server {
       if (this.config.allowIntrospection === true) {
         throw new IntrospectionNotPermitted('allowListPath')
       }
+      if (this.config.maxQueryComplexity) {
+        throw new QueryComplexityLimitationNotPermitted('allowListPath')
```

Comment:
Is there a reason why we are preventing server from running when this option is used with allowListPath. Couldn't we just log a warning, discard this value and go on?

Reply:
I like to be concrete about intent with config. If we do not throw, then the user could easily miss a warning and then have unexpected results.